apps/data-pipeline/course-scraper/src/index.ts

```diff
@@ -689,6 +695,8 @@ async function main() {
   logger.info("Running I&C SCI 32A/H32 shim...");
   await patchH32({ db });
   logger.info("All done!");
+
+  await db.refreshMaterializedView(courseView);
```

Comment:
This should run before the last print, so that "All done!" actually means what it says.